tarball/archive.go

```diff
@@ -0,0 +1,175 @@
+package tarball
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"context"
+	"io"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	fspkg "github.com/Scalingo/go-utils/fs"
+	"github.com/Scalingo/go-utils/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+var EmptyArchiveError = errors.New("empty archive")
```

Comment:
```suggestion
var ErrEmptyArchive = errors.New("empty archive")
```

Error variables start with `Err` (https://github.com/golang/go/wiki/Errors#naming) :grin:

Reply:
deleted: was not used